README.md

```diff
@@ -1,70 +1,60 @@
 
 You will also need to create a Discord bot and obtain a token. Follow the official Discord developer documentation to : [Creating a Bot Account](https://discordpy.readthedocs.io/en/stable/discord.html).
 
 # Setup
 
 Clone or download this repository to your local machine.
 
-Install the necessary Python libraries. You can do this by manually installing each library your project depends on using pip.
-
-Create a .env file in the project directory and add the following environment variables:
-
-    DISCORD_TOKEN=<Your Discord Bot Token>
-    BOT_CREATOR_ID=<Your Discord User ID>
-
-# Configuration
-
-You can customize the bot's behavior by modifying the following global variables in the code:
+Install the necessary node modules by using: "npm install" in your desired command line.
 
-    STICKER_CHAIN_LIMIT: The maximum number of consecutive messages with stickers before stopping the cleaning process.
+Create a config.json file in the project directory and add the following keys + values:
 
-    CLEAN_INTERVAL_SECONDS: The interval in seconds for checking and cleaning messages with stickers.
+	"token": <YOUR TOKEN HERE>,
+	"clientId": <YOUR CLIENT ID HERE>,
+	"guildId": <YOUR GUILD ID HERE>,
+	"userId": <YOUR USER ID HERE>,
+	"roleId": <ROLE ID HERE>
 
-    DELETE_MESSAGE_TIME: The time in seconds to delete response messages automatically.
+**NOTE** The creator and role ID is used for permission validation with commands.
 
-    MESSAGE_HISTORY_LIMIT: The maximum number of messages to check during cleaning.
+If you want any user with a certain role to be able to use the bot, use role ID. If you want to be the only one to use the bot, edit the if/else statement within "client.on" (line 49-53) function in index.js as such:
 
-    NUKE_LIMIT: The maximum number of messages to nuke when using the $1984 command.
-
-    ASYNC_SLEEP: The delay in seconds between message deletions during cleaning and nuking.
-
-    COMMAND_PREFIX: The prefix used for bot commands.
+```
+    if (interaction.member.id === userId) {
+        await command.execute(interaction);
+    } else {
+        await interaction.reply("It seems you don't have permission...")
+    }
+```
 
 # Bot Commands
-    $addbl [message_id]: Add a message to the blacklist to prevent $1984 from deleting it.
-    $removebl [message_id]: Removes a message from the blacklist, allowing $1984 to delete it.
-    $clearbl: Clear's the blacklist.
-    $clean [user_id] [max_count]: Deletes a specified number of messages with stickers sent by a user.
-    $hello: Greets the user with a simple message.
-    $health: Checks if the bot is operational and displays its latency.
-    $1984 [user_id] [max_count]: "Nukes" a specified number of messages from a user by deleting them.
-    $ping: Checks the bot's latency.
+    /addbl [message_id]: Add a message to the blacklist to prevent $1984 from deleting it.
```

Comment:
Feel free to disagree, but if we alphabetize it 

```suggestion
/1984 [user_id] [max_count]: "Nukes" a specified number of messages from a user by deleting them.
/addbl [message_id]: Add a message to the blacklist to prevent $1984 from deleting it.
/clean [user_id] [max_count]: Deletes a specified number of messages with stickers sent by a user.
/clearbl: Clear's the blacklist.
/health: Checks if the bot is operational and displays its latency.
/hello: Greets the user with a simple message.
/removebl [message_id]: Removes a message from the blacklist, allowing $1984 to delete it.
```